docs/hub/webhooks-guide-auto-retrain.md

```diff
@@ -0,0 +1,176 @@
+# Webhook guide: Setup an automatic system to re-train a model when a dataset changes
+
+This guide will guide you through the set up of an automatic training pipeline on the HuggingFace platform,
+using HF Webhooks, Spaces and AutoTrain.
+
+We will build a Webhook that listen to changes on an image classification dataset, and triggers a fine-tuning
+of [microsoft/resnet-50](https://huggingface.co/microsoft/resnet-50) using [AutoTrain](https://huggingface.co/autotrain).
+
+
+
+## Pre-requisite: upload your dataset to the Hub
+
+We will use a [simple image classification dataset](https://huggingface.co/datasets/sbrandeis-test-org/input-dataset) for the sake
```

Comment:
I wonder if it's worth it to put them in some special org (maybe https://huggingface.co/huggingface-projects ?)

Reply:
(personal repos is fine for guides and doc IMO, gives a more personal touch)

Reply:
(but no need to switch back...)